docs/administration.md

```diff
@@ -0,0 +1,49 @@
+# Administration
+
+You can acces uMap administration page by navigating to  `https://your.server.org/admin`
+
+You will have to connect with the admin account create during installation. Default admin username is umap.
+
+## Pictograms
+
+Pictograms are icon that can be used in your map markers.
+
+Pictograms are not embedded in uMap sources, you will have to add them manually. So you can choose which pictograms you want to use.
+
+Example of pictograms libraries you may want to use:
+
+- [Maki Icons](https://labs.mapbox.com/maki-icons/) (icon set made for map designers)
+- [Osmic Icons](https://gitlab.com/gmgeo/osmic)
+- [SJJB Icons](http://www.sjjb.co.uk/mapicons/contactsheet)
+
+### Import pictograms
+
+To import pictograms on your uMap server, you will need to use command `umap import_pictograms`
+
+Note, you can get help with `umap import_pictograms -h`
+
+In this example, we are gonna import Maki Icons.
+
+First, we download icons from main site. Inside the downloaded archive, we keep only the icons folder that contains svg files. Place this folder on your server.
```

Comment:
contain ?

Reply:
It contains files. "s" because this is the third singular form 😉